src/utils/filteredTodos.ts

```diff
@@ -0,0 +1,22 @@
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  filterType: FilterType,
+): Todo[] => {
+  const preparedTodos = todos.filter(todo => {
+    switch (filterType) {
+      case FilterType.All:
+        return true;
+      case FilterType.Active:
+        return !todo.completed;
+      case FilterType.Completed:
+        return todo.completed;
+      default:
```

Comment:
you have typization, so no need to handle case of incorrect type, as for me `All` case is default

Reply:
ready 🚀